php-nginx/whitelist_functions.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+$phpDir = getenv("PHP_DIR");
+
+if ($phpDir === false) {
+    exit(0);
+}
+
+$whitelist = getenv("WHITELIST_FUNCTIONS");
+
+if ($whitelist === false) {
+    exit(0);
+} else {
+    $whitelist = array_filter(array_map('trim', explode(',', $whitelist)));
+}
+
+$suhosinBlacklist = ini_get('suhosin.executor.func.blacklist');
+$disable_functions = ini_get('disable_functions');
+
+$whitelist_ini = '';
+
+if ($suhosinBlacklist !== false) {
```

Comment:
So `WHITELIST_FUNCTIONS` trumps `suhosin.executor.func.blacklist` and `disable_functions`. I think this makes sense.